astro/src/content/quickstarts/quickstart-react-native.mdx

```diff
@@ -0,0 +1,208 @@
+
+```shell
+npx create-expo-app my-react-native-app && cd my-react-native-app
+```
+
+<Aside type="note">
+  If this is your first time setting up a React Native application, you'll receive a message asking if you want to install `create-expo-app`, so press `y` to confirm.
+</Aside>
+
+## Authentication
+
+We'll use the [Expo AuthSession library](https://docs.expo.dev/versions/latest/sdk/auth-session/), which simplifies integrating with FusionAuth and creating a secure web application.
+
+### Configure Expo AuthSession
+
+Install `expo-auth-session`, its dependency `expo-crypto` to handle cryptographic operations and `expo-web-browser` to interact with the device Browser in order to log the user in or out.
+
+```shell
+npx expo install expo-auth-session expo-crypto expo-web-browser
+```
+
+Create a `.env` file to hold information about your FusionAuth instance and application, replacing the value in `EXPO_PUBLIC_FUSIONAUTH_URL` with the address you copied from `ngrok` when [exposing your instance](#expose-your-instance).
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/.env"
+            lang="ini"/>
+
+Edit `app.json` to add a `scheme` that will be used when redirecting back to your application after logging in and out of FusionAuth.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/app.json"
+            lang="json"/>
+
+## App Customization
+
+In this section, you'll turn your application into a trivial banking application with some styling.
+
+### Add Styling
+
+First, run the command below to install some libraries needed for theming.
+
+```shell
+npm install expo-image expo-constants react-native-currency-input
+```
+
+Instead of using CSS, React Native has its own concept of [stylesheets](https://reactnative.dev/docs/stylesheet). Create a file named `changebank.style.js` with the contents below to style your ChangeBank app.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/changebank.style.js"
+            lang="javascript"/>
+
+## Finish Setting up the App
+
+Change the existing `App.js` to integrate `expo-auth-session`, add the ChangeBank template and stitch up everything.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/App.js"
+            lang="javascript"/>
+
+## Run the App
+
+If you want to test on an Android device, connect a physical device via USB or install [Android Studio](https://developer.android.com/studio) to set up an emulator.
+
+If you want to test on an iOS device and are running a Mac, install [Xcode](https://developer.apple.com/xcode/) to set up an emulator.
+Start up the React Native application using this command:
+
+```shell
+npx expo start
```

Comment:
I was getting some warnings but I went back and updated the app.json to more match your example and it loos like the errors went away.

Reply:
Just realized this could be due to my environment setup.  Need to get docker working.